src/lib/LeafletMeasure.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import L from 'leaflet';
+import { MapControl, withLeaflet } from 'react-leaflet';
+
+
+require('leaflet-measure')
+require('leaflet-measure/dist/leaflet-measure.css')
+
+
+export class LeafletMeasure extends MapControl {
+
+  createLeafletElement(props) {
+    var options = {}
```

Comment:
If you add default values for the parameters, this step (stripping empty args) might not be necessary.